### ISSUE TITLE
TypeError: Cannot read property 'avatar_url' of null

### ISSUE THREAD

**Issue by Shaun-Harrison** [toko-bifrost/ms-teams-deploy-card#11]
The github action is being marked as completed, but no message is appearing in teams

The steps is showing the following error

```
Post job cleanup.
Workflow run information: {
  "owner": "wcbs-ltd",
  "repo": "hub-ui",
  "ref": "3d36a7328a26420e1578759ce60f89d6ab9443d2",
  "branchUrl": "https://github.com/wcbs-ltd/hub-ui/tree/refs/heads/GithubActions",
  "runId": "131963184",
  "runNum": "2507"
}
(node:3692) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'avatar_url' of null
    at Object.formatCozyLayout (/home/runner/work/_actions/toko-bifrost/ms-teams-deploy-card/3.0.1/dist/post/index.js:1:365095)
    at Object.formatCompleteLayout (/home/runner/work/_actions/toko-bifrost/ms-teams-deploy-card/3.0.1/dist/post/index.js:1:4207)
    at Object.formatAndNotify (/home/runner/work/_actions/toko-bifrost/ms-teams-deploy-card/3.0.1/dist/post/index.js:1:16527)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(node:3692) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:3692) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by aldwyn-acn** [comment]
Hi @Shaun-Harrison, may I know what type of git action it is trying to process? For instance, `push`, `pull_request`?

**Comment by Shaun-Harrison** [comment]
Hello, this was a push

**Comment by aldwyn-acn** [comment]
I understand now that both `author` and `committer` will be `null` if the committer is not associated with a Github user. https://github.community/t/request-for-commits-author-null-and-committer-null-cant-get-user-login/14409/2. I guess I'll just have to remove the author details in the card if its null.

**Comment by aldwyn-acn** [comment]
This has been resolved on #12.
